extensions/vertx-http/deployment/src/test/java/io/quarkus/vertx/http/devmode/ArcEndpointTest.java

```diff
@@ -46,21 +50,28 @@ public void testBeans() {
 
         for (int i = 0; i < observers.size(); i++) {
             JsonObject observer = observers.getJsonObject(i);
-            if (beanId.equals(observer.getString("declaringBean"))) {
+            if (beanId.equals(observer.getString("declaringBean"))
+                    && StartupEvent.class.getName().equals(observer.getString("observedType"))) {
                 fooObserver = observer;
+                assertEquals(2500, fooObserver.getInteger("priority"));
             }
         }
         assertNotNull(fooObserver);
-        assertEquals(StartupEvent.class.getName(), fooObserver.getString("observedType"));
-        assertEquals(2500, fooObserver.getInteger("priority"));
     }
 
     @Named
     @ApplicationScoped
     public static class Foo {
 
+        @Inject // do not remove the bean
+        HttpBuildTimeConfig httpConfig;
```

Comment:
I'm a little unclear about this, can you please explain it?

Reply:
Ah, that's a leftover from the previous version of the test where I tried to inject the config root using `Arc.container().instance()` (and it's removable). Will remove the comment...

Reply:
Cool, because I was beginning to worry :)